include/deal.II/matrix_free/shape_info.h

```diff
@@ -467,6 +467,14 @@ namespace internal
        */
       dealii::Table<2, unsigned int> face_to_cell_index_hermite;
 
+      /**
+       * For degrees on faces, the basis functions are not
+       * in the correct order if a face is not in the standard orientation
+       * to a given element. This data structure is used to re-order the
+       * basis functions to represent the correct order.
+       */
+      dealii::Table<2, unsigned int> face_orientations;
```

Comment:
I guess we should remove the array from the mapping info class?

Reply:
It is still used in `integrate_scatter`, `scatter`, and `evalute`

Reply:
I was more thinking about the future when we have everything. Right now it's clear we need it this way.